clang/lib/CodeGen/TargetInfo.cpp

```diff
@@ -4427,6 +4469,10 @@ ABIArgInfo WinX86_64ABIInfo::classify(QualType Ty, unsigned &FreeSSERegs,
 }
 
 void WinX86_64ABIInfo::computeInfo(CGFunctionInfo &FI) const {
```

Comment:
Can you extend the test below for this target as well?

Reply:
triple x86_64-unknown-unknown in the test is already covering WinX86_64ABIInfo

Reply:
Is that because the test is being run on Windows?  Otherwise, I am not sure how it defaults to Windows and not say, for example, Linux.

Reply:
> Is that because the test is being run on Windows?

Yes. On linux, the test runs in X86_64ABIInfo::computeInfo path.

Reply:
Okay, please add a Windows-specific triple to the test.

Reply:
> triple x86_64-unknown-unknown in the test is already covering WinX86_64ABIInfo

sorry this answer is incorrect. You're right that windows x86_64 target isn't covered in the first commit. I've added WinX86_64ABIInfo test in the new commit.

I also changed i686-unknown-unknown triple to i686-pc-win32-gnu, in order to match with the use scenario of this PR in OpenCL builtin build. DEVICE_TRIPLE is i686-pc-win32-gnu-elf and x86_64-pc-win32-gnu-elf for windows build in file [backend/libraries/CMakeLists.txt](https://github.com/intel/llvm/pull/8216/files#diff-ab2b2e6aaad4bf10bc0972e24d5251fa6868588836e19001dd34a0dd4c3c0057)